src/ripple/app/misc/Validations.cpp

```diff
@@ -96,6 +96,14 @@ class ValidationsImp : public Validations
         if (!val->isTrusted() && pubKey)
             val->setTrusted();
 
+        // Do not process partial validations.
+        if(!val->isFull())
+        {
+            // Only forward if current
+            return isCurrent;
```

Comment:
@JoelKatz suggested we have this match how we treat full validations, only forwarding if current and trusted:
```
return isCurrent && val->isTrusted();
```